src/display.jl

```diff
@@ -133,6 +146,9 @@ function subscriptify(n::Int)
 end
 
 
+# fall-back:
```

Comment:
I had just `representation` before. The problem is that `representation(x::Interval{BigFloat})` calls `representation` for the interval part and then adds the part with the number of digits. 

To do so, I had to use `invoke`, which is ugly. The simple solution was to add `basic_representation` instead.